pages/dkp/kommander/2.2/licensing/enterprise/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: layout.pug
+navigationTitle: DKP Enterprise
+title: DKP Enterprise
+excerpt: A section containing all of the features available to users with a DKP Enterprise license.
+menuWeight: 5
+beta: false
+---
+
+## DKP Enterprise overview
+
+![DKP Enterprise Diagram](../img/dkpenterprisediagram.png)
+
+DKP Enterprise is a multi-cluster lifecycle management Kubernetes solution centered around a FIPS compliant management cluster that manages multiple attached or managed Kubernetes clusters through a centralized management dashboard that gives you a single point of observability and control throughout all of your attached or managed clusters. The DKP Enterprise license gives the user access to the entire Konvoy cluster environment, the Kommander UI dashboard that deploys platform and catalog applications as well as multi-cluster management, and comprehensive compatibility with our full range of infrastructure deployment options.
+
+## Compatible infrastructure
+
+DKP Enterprise operates across D2iQ's entire range of cloud, on-premise, edge, and air-gapped infrastructures and has support for various OSs, including immutable OSs.
+
+Deployment options for DKP Enterprise include:
+
+* Bare metal servers
+* VMware vSphere   
+
+Cloud deployment options for DKP Enterprise include:
```

Comment:
I'm thinking we should NOT list these here and just point to the OS table that already talks about all the combinations we officially support:

https://docs.d2iq.com/dkp/konvoy/2.1/supported-operating-systems/

Reply:
Sounds good, so per @cmastr recommendation @cneth would a link to currently supported OS table be the solution here?

Reply:
Removed section and linked to https://docs.d2iq.com/dkp/konvoy/2.1/supported-operating-systems/.